packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -107,20 +111,28 @@ export default class {
   }
 
   async getAllRemotesInfo() {
-    const remotes = await this._remotes.get()
+    const remotesInfo = this._remotesInfo
+    await asyncMap(this._remotes.get(), async remote => {
+      const getInfo =
+        remote.proxy !== undefined
+          ? () =>
+              this._xo.callProxyMethod(remote.proxy, 'remote.getInfo', {
+                remote,
+              })
+          : await this.getRemoteHandler(remote.id).then(
+              handler => ::handler.getInfo
```

Comment:
This syntax is deprecated and will never be officialized by ECMAScript, let's avoid it in new code.